src/table/pagination-table/components/body/TableBodyWrapper.tsx

```diff
@@ -95,7 +96,7 @@ const TableBodyWrapper = ({ setShouldRefocus, tableWrapperRef, announce }: Table
                     handleMouseDownToFocusBody(cell, rootElement, setFocusedCellCoord, keyboard, totalsPosition)
                   }
                 >
-                  <CellText fontSize={cellStyle.fontSize}>{cell.qText}</CellText>
+                  <CellText fontSize={cellStyle.fontSize} lines={viewService.viewState?.maxLineCount || 3}>{cell.qText}</CellText>
```

Comment:
`CellText` already have a default value of 3, no need to pass it

Reply:
I want to avoid the case when `maxLineCount` is set to 0

Reply:
But I guess we can ignore that case.